nurseconnect/tests/test_profiles.py

```diff
@@ -104,6 +98,32 @@ def test_register_user_validation(self):
             [u"Username already exists."]
         )
 
+    def test_invalid_clinic_code_raises_error(self):
```

Comment:
Are you not accessing the view, which renders the form, which calls `check_clinic_code(clinic_code)`, which does the API call?